src/backend/gl/src/command.rs

```diff
@@ -32,6 +32,7 @@ fn primitive_to_gl(primitive: c::Primitive) -> gl::types::GLenum {
         TriangleList => gl::TRIANGLES,
         TriangleStrip => gl::TRIANGLE_STRIP,
         //TriangleFan => gl::TRIANGLE_FAN,
+        QuadList => gl::PATCHES
```

Comment:
Note to self: don't forget to call `glPatchParameteri​` somewhere, not to mention the need to fix the GL backend to support tessellation properly and run the new example ;)